mopidy_alsamixer/__init__.py

```diff
@@ -22,6 +22,9 @@ def get_config_schema(self):
         schema = super(Extension, self).get_config_schema()
         schema['card'] = config.Integer(minimum=0)
         schema['control'] = config.String()
+        schema['volmin'] = config.Integer(minimum=0, maximum=100)
+        schema['volmax'] = config.Integer(minimum=0, maximum=100)
+        schema['logarithmic_volume'] = config.Boolean()
```

Comment:
Should we make this `config.String(choices=('linear', 'logarithmic'))` instead? In case we need to or want to add cubic at some point. Option would be `volume_format` or perhaps `volume_scale`?